Sources/VaporAuth/Provider.swift

```diff
@@ -0,0 +1,13 @@
+import Vapor
+
+public final class Provider: Vapor.Provider {
```

Comment:
do we still want people to add provider in case of future behavior? (is that what this is for?)